rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/interpreter/move_a.kt

```diff
@@ -95,6 +95,15 @@ private fun InterpreterCore.toArray(expression: Expression): ConcreteArrayValue
                 ConcreteArrayValue(mutableListOf(eval(expression)), expression.type())
             }
         }
+        is IntLiteral -> {
```

Comment:
What happens if type is RealLiteral

Reply:
While I was creating the tests, I tried this one too:
```
     D A146            S             14  6 DIM(9) INZ
     C                   MOVEA     123.5         A146
```
but didn't compile. Because the domain was for `MOVEA`, I didn't try for another operators.